### PR TITLE
Kp unused imports

### PR DESCRIPTION
**Pull request by keithasaurus**
Just cleaned up a bunch of imports in the mypy package and did a tiny bit of code housekeeping. All tests pass.

### CODE REVIEW

**Comment by gvanrossum** [comment]
That's a lot of churn. Did you use some kind of tool to do this?

**Comment by keithasaurus** [comment]
Yeah, code inspection on PyCharm is pretty outstanding.

**Comment by gvanrossum** [comment]
I'm okay with killing dead code but not with moving the imports around.

On Thu, Feb 25, 2016 at 4:10 PM, keithasaurus <notifications@github.com>
wrote:

> Yeah, code inspection on PyCharm is pretty outstanding.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python/mypy/pull/1244#issuecomment-189046892>.
>



-- 
--Guido van Rossum (python.org/~guido)

**Comment by keithasaurus** [comment]
Just curious, what's the reasoning?

File: mypy/checkexpr.py [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
-    TupleType, Instance, TypeVarType, TypeTranslator, ErasedType, FunctionLike, UnionType,
-    PartialType, DeletedType
-)
+import mypy.checker
+from mypy import applytype
+from mypy import erasetype
+from mypy import join
+from mypy import messages
+from mypy import nodes
+from mypy import types
```
Comment (kirbyfan64) [inline mypy/checkexpr.py]: Side note: Everywhere else, multiple names are imported in one line, like:

```python
from mypy import erasetype, join, messages, nodes, types
```

Should that be done here?

Comment (keithasaurus) [inline mypy/checkexpr.py]: My opinion would be yes, but if Guido doesn't want to move around the imports, that's a moot point.

**Comment by gvanrossum** [comment]
The import reorgs just add noise to the history of the files without any benefit. When we adopt a style guide for mypy itself and decide to clean up all the style violations we can reconsider. But at this stage I vote for not polluting things like git blame output.

**Comment by keithasaurus** [comment]
Gotcha. I'll scale back and make a clean pull request with only dead code changes.

**Pull request closed without merging**